facebook/OpenBIC
# fby3.5: bb: Support get MB index

fby3.5: bb: Support get MB index
Summary:
- Support getting motherboards index.

Dependency: #241 

Test Plan:
- Build code: Pass
- Get MB index: Pass

Log:
1. Check can get MB index successful.
- Slot 1
root@bmc-oob:~# bic-util slot1 0xE0 0x02 0x9C 0x9C 0x00 0x10 0xC0 0xF0
9C 9C 00 10 31 F0 00 01

- Slot 3
root@bmc-oob:~# bic-util slot1 0xE0 0x02 0x9C 0x9C 0x00 0x10 0xC0 0xF0
9C 9C 00 10 31 F0 00 03
